### ISSUE TITLE
server crash

### ISSUE THREAD

**Issue by MSandro** [Direwolf20-MC/BuildingGadgets#236]
forge-1.12.2-14.23.5.2808-universal
BuildingGadgets-2.6.2

```
Time: 2/12/19 10:18 PM
Description: Exception in server tick loop

java.lang.NoClassDefFoundError: net/minecraft/client/audio/ISound
	at com.direwolf20.buildinggadgets.common.proxy.CommonProxy.registerSounds(CommonProxy.java:99)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_84_CommonProxy_registerSounds_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:814)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:628)
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99)
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.audio.ISound
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 12 more
```

**Comment by MajorTuvok** [comment]
For future reference this was fixed in 2.6.3

**Comment by Phylogeny** [comment]
Duplicate of #233
